sdk/src/main/scala/com/horizen/certificatesubmitter/CertificateSubmitter.scala

```diff
@@ -91,9 +89,12 @@ class CertificateSubmitter
     val actualSysDataConstant =
       CryptoLibProvider.sigProofThresholdCircuitFunctions.generateSysDataConstant(signersPublicKeys.map(_.bytes()).asJava, params.signersThreshold)
 
-    val expectedSysDataConstant = genSysConstant//getSidechainCreationTransaction(sidechainNodeView.history).getBackwardTransferPoseidonRootHash
+    val expectedSysDataConstant = params.calculatedSysDataConstant
```

Comment:
I think, that here we can use `params.calculatedSysDataConstant` directly, same as in State.

Reply:
changed